### PR TITLE
Update to Metal 5.0.0

### PR DESCRIPTION
**Pull request by TheUnknownCylon**

### CODE REVIEW

File: metal-tools-formats/src/test/java/nl/gertjanal/metaltools/formats/rar/RARTest.java [hunk 1]
```diff
@@ -20,6 +21,7 @@ public void testRAR4() throws Exception {
 	}
 
 	@Test
+	@Ignore
```
Comment (gertjanal) [inline metal-tools-formats/src/test/java/nl/gertjanal/metaltools/formats/rar/RARTest.java]: ?

**Comment by gertjanal** [comment]
awesome. Thnx

File: metal-tools-formats/src/test/java/nl/gertjanal/metaltools/formats/rar/RARTest.java (hunk 1 above)
Comment (TheUnknownCylon) [inline metal-tools-formats/src/test/java/nl/gertjanal/metaltools/formats/rar/RARTest.java]: Test file is missing in the repository.

```
[I] ~/p/m/metal-tools (master ⚡=) ls ./metal-tools-formats/src/test/resources/rar
example4.x.rar
```

File: metal-tools-jshexviewer/src/main/java/nl/gertjanal/metaltools/jshexviewer/JsHexViewer.java [hunk 2]
```diff
@@ -154,7 +155,9 @@ private static void step(final ParseItem item, final Map<Long, LinkedList<Defini
 		if (!item.isGraph()) {
 			if (item.getDefinition() instanceof Def) {
 				final ParseValue value = item.asValue();
-				getList(map, new Definition(value));
+				if(value.slice.source instanceof ByteStreamSource) {
+					getList(map, new Definition(value));
+				}
```
Comment (jvdb) [inline metal-tools-jshexviewer/src/main/java/nl/gertjanal/metaltools/jshexviewer/JsHexViewer.java]: So the only thing needed here for full `Tie`-support is implementing the `else`-clause :-)

Comment (gertjanal) [inline metal-tools-jshexviewer/src/main/java/nl/gertjanal/metaltools/jshexviewer/JsHexViewer.java]: But a tie means we won't find any bytes in the read data, right?

File: metal-tools-formats/src/test/java/nl/gertjanal/metaltools/formats/rar/RARTest.java (hunk 1 above)
Comment (gertjanal) [inline metal-tools-formats/src/test/java/nl/gertjanal/metaltools/formats/rar/RARTest.java]: oops

**Pull request merged**